vendors/nordic/boards/nrf52840-dk/ports/common_io/iot_test_common_io_internal.c

```diff
@@ -29,6 +29,19 @@
 
 #include "iot_test_common_io_internal.h"
 
+#if defined( IOT_TEST_COMMON_IO_GPIO_SUPPORTED ) && ( IOT_TEST_COMMON_IO_GPIO_SUPPORTED >= 1 )
```

Comment:
`IOT_TEST_COMMON_IO_GPIO_SUPPORTED  >= 2` ?

Reply:
Thanks. Implemented in latest commit.